plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -9007,6 +9007,26 @@ public void testSelectWithShortZoneId()
         }
     }
 
+    @Test
+    public void testOrcUseColumnNames()
```

Comment:
some coverage is already provided by testReadWithPartitionSchemaMismatch and TestHivePartitionSchemaEvolution.

let's make sure the new test adds new value